pkg/k8shandler/kibana/reconciler.go

```diff
@@ -64,12 +68,17 @@ func Reconcile(requestCluster *kibana.Kibana, requestClient client.Client, esCli
 		return err
 	}
 
-	if err := clusterKibanaRequest.createOrUpdateKibanaConsoleExternalLogLink(); err != nil {
-		return err
-	}
+	// we only want to create these if the use case is the CLO one
+	// make sure our namespace is "openshift-logging" and our cr name is "kibana"
+	// or do we just check that our owner ref is from a cluster logging object?
+	if clusterKibanaRequest.isClusterLoggingOperatorCreated() {
```

Comment:
Suggest to use the same terminology as k8s here. It is literaly created, but we check if the `ownerReference` equals CLO. WDYT renaming this to `hasOwnerReference(ClusterLoggingOwnerReference{})` or `ownedBy(ClusterLoggingOwnerReference{})`?

Reply:
the only reason i avoided that was because we have the additional check if it matches a particular name and namespace (for the cases where we are testing just the elasticsearch operator without clo)